package.json

```diff
@@ -1,40 +1,47 @@
 {
-  "name": "matrix-react-sdk",
+  "name": "vector-web",
   "version": "0.0.1",
-  "description": "SDK for matrix.org using React",
+  "description": "Vector webapp",
   "author": "matrix.org",
   "repository": {
     "type": "git",
-    "url": "https://github.com/matrix-org/matrix-react-sdk"
+    "url": "https://github.com/vector-im/vector-web"
   },
   "license": "Apache-2.0",
-  "main": "src/index.js",
   "style": "bundle.css",
   "scripts": {
-    "build:skins": "jsx skins build/skins",
-    "build:logic": "jsx src build/src",
-    "build:js": "npm run build:skins && npm run build:logic",
-    "start:js": "jsx -w skins/base/views/ build --source-map-inline",
-    "build:css": "catw 'skins/base/css/**/*.css' -o bundle.css -c uglifycss --no-watch",
-    "start:css": "catw 'skins/base/css/**/*.css' -o bundle.css -v",
-    "build": "npm run build:js && npm run build:css",
-    "start": "parallelshell \"npm run start:js\" \"npm run start:css\"",
-    "prepublish": "npm run build"
+    "reskindex": "reskindex vector -h src/skins/vector/header",
+    "build:css": "catw 'src/skins/vector/css/**/*.css' -o vector/bundle.css -c uglifycss --no-watch",
+    "build:compile": "babel --source-maps -d lib src",
+    "build:bundle": "NODE_ENV=production webpack -p lib/vector/index.js vector/bundle.js",
+    "build": "npm run build:css && npm run build:compile && npm run build:bundle",
+    "start:js": "webpack -w src/vector/index.js vector/bundle.js",
+    "start:skins:css": "catw 'src/skins/vector/css/**/*.css' -o vector/bundle.css",
+    "//cache": "Note the -c 1 below due to https://code.google.com/p/chromium/issues/detail?id=508270",
+    "start": "parallelshell 'npm run start:js' 'npm run start:skins:css' 'http-server -c 1 vector'",
```

Comment:
Single-quotes like this will prevent this from working on Windows. It needs to be `\"`.